hack/upgrade-test.sh

```diff
@@ -181,8 +181,12 @@ echo "----- Images before upgrade"
 ${CMD} get deployments -n kubevirt-hyperconverged -o yaml | grep image | grep -v imagePullPolicy
 ${CMD} get pod $HCO_CATALOGSOURCE_POD -n ${HCO_CATALOG_NAMESPACE} -o yaml | grep image | grep -v imagePullPolicy
 
+echo "-- check that cluster is operational"
```

Comment:
Can you move lines 184-185 to right after line 175?

Reply:
done.

Reply:
done